src/interpreter.ts

```diff
@@ -388,6 +388,18 @@ export class Interpreter<
     });
     return this;
   }
+
+  /**
+   * Starts the interpreter keeping the current process running
+   * @param initialState The state to start the statechart from
+   */
+  public run(initialState?: State<TContext, TEvent> | StateValue): Interpreter<TContext, TStateSchema, TEvent> {
+      if (!this.initialized) {
+        this.start(initialState);
+      }
+      setImmediate(this.idle.bind(this));
```

Comment:
This won't work for browser environments, by the way

Reply:
Updated this to use setTimeout(..., 0) instead.